### ISSUE TITLE
Acunetix - Git repository found

### ISSUE THREAD

**Issue by payloadh** [payloadh/test#3]
<table style="border:1px solid #ccc"><tr><td style="border-right:1px solid #ccc;border-bottom:1px solid #ccc; padding-right: 5px;"><strong>Target URL</strong></td><td style="border-bottom:1px solid #ccc">https://eazytraining.fr/</td></tr><tr><td style="border-right:1px solid #ccc;"><strong>Severity</strong></td><td>High</td></tr></table><h5>Affects</h5>https://eazytraining.fr/
<h5>Attack Details</h5>Git files found at : <span class="bb-dark">/.git/config</span><br/><br/> Repository files/directories: <br/> <ul>      <li>.DS_Store</li>      <li>.htaccess</li>      <li>index.php</li>      <li>license.txt</li>      <li>readme.html</li>      <li>wp-activate.php</li>      <li>wp-admin/about.php</li>      <li>wp-admin/admin-ajax.php</li>      <li>wp-admin/admin-footer.php</li>      <li>wp-admin/admin-functions.php</li>      <li>...</li>  </ul>
<h5>HTTP Request</h5>	GET /.git/config HTTP/1.1
	Cookie: _learn_press_session_59a5f4789e4d751529a84bd17fc11cbf=aa403507fbb83fcb0e315b2d80461901%7C%7C1636477656%7C%7C0046c3d18484cbebde919fc6dbee48f8;_wordpress_lp_guest=901769aeb191f60809e6f587879f5076;pmpro_visit=1
	Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
	Accept-Encoding: gzip,deflate
	Host: eazytraining.fr
	User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.61 Safari/537.36
	Connection: Keep-alive
	
	
<h5>Vulnerability Description</h5>Git metadata directory (.git) was found in this folder. An attacker can extract sensitive information by requesting the hidden metadata directory that version control tool Git creates. The metadata directories are used for development purposes to keep track of development changes to a set of source code before it is committed back to a central repository (and vice-versa). When code is rolled to a live server from a repository, it is supposed to be done as an export rather than as a local working copy, and hence this problem.
<h5>Impact</h5>These files may expose sensitive information that may help an malicious user to prepare more advanced attacks.
<h5>Remediation</h5>Remove these files from production systems or restrict access to the .git directory. To deny access to all the .git folders you need to add the following lines in the appropriate context (either global config, or vhost/directory, or from .htaccess):
<code><pre>
&lt;Directory ~ &quot;\.git&quot;&gt;
Order allow,deny
Deny from all
&lt;/Directory&gt;
</pre></code>
<hr><h5>References:</h5><a href=http://www.ducea.com/2006/08/11/apache-tips-tricks-deny-access-to-some-folders/>Apache Tips & Tricks: Deny access to some folders</a><br/><br/>
